sympy/functions/elementary/tests/test_exponential.py

```diff
@@ -138,13 +143,19 @@ def test_exp_taylor_term():
     assert exp(x).taylor_term(1, x) == x
     assert exp(x).taylor_term(3, x) == x**3/6
     assert exp(x).taylor_term(4, x) == x**4/24
+    assert exp(x).taylor_term(-1, x) == S.Zero
 
 
 def test_exp_MatrixSymbol():
     A = MatrixSymbol("A", 2, 2)
     assert exp(A).has(exp)
```

Comment:
Maybe this test should use unchanged from sympy.core.expr

Reply:
I created #16997 instead... Although this is not covered there.